doc/source/examples/pd_variance_regression_friedman.ipynb

```diff
@@ -0,0 +1,406 @@
+{
```

Comment:
<p>we exposed -&gt; we expose (generally use present tense for describing functionality)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>